app/controllers/data_api/tad_data_exports_controller.rb

```diff
@@ -2,10 +2,58 @@ module DataAPI
   class TADDataExportsController < ActionController::API
     include ServiceAPIUserAuthentication
 
+    def index
+      exports = DataAPI::TADExport.all
+
+      formatted_exports = exports.map do |e|
+        {
+          export_date: e.completed_at,
+          description: e.name,
+          url: data_api_tad_export_url(e.id),
+        }
+      end
+
+      render json: { data: formatted_exports.as_json }
+    end
+
+    def show
+      data_export = DataAPI::TADExport.all.find(params[:id])
+      serve_export(data_export)
+    end
+
     def latest
       data_export = DataAPI::TADExport.latest
-      data_export.update!(audit_comment: "File downloaded via API using token ID #{@authenticating_token.id}")
-      send_data data_export.data, filename: data_export.filename
+      serve_export(data_export)
+    end
+
+  private
+
+    def serve_export(export)
+      export.update!(audit_comment: "File downloaded via API using token ID #{@authenticating_token.id}")
+      send_data export.data, filename: export.filename
+    end
+
+    def verify_token!
```

Comment:
this can be removed as it's part of the `ServiceAPIUserAuthentication` module which is included in the controller

Reply:
Done